### DISCUSSION TITLE
Cant find database name

### DISCUSSION THREAD

**Discussion by LebTeb** [vrtmrz/obsidian-livesync#253] [Q&A]
Hello!

I was wondering if i could be shown where the database name is located?
in `settings.md` it told me it should be made automatically but I was never told what it should be called.
![image](https://github.com/vrtmrz/obsidian-livesync/assets/101553924/8d953ec8-c6ee-4b65-930b-89d5463ebc95)

Thanks
Leb

**Comment by vrtmrz** [comment]
Thank you for asking me! and sorry for being late!
`the_database` is the database name, in this screenshot.
If we could connect to the remote database (Which is serving at the `URI`, and could accept `Username` and `Password`), the database which is named `the_database` should be created automatically in the remote database.
The actual URL might be [URI]/[Database name].

I hope this answers your question.

**Comment by LebTeb** [comment]
I'm not exactly sure what I changed but I am now getting a new error. 
![image](https://github.com/vrtmrz/obsidian-livesync/assets/101553924/fca7c937-724b-429a-9adb-46c0dd07ac04)

When I run `docker ps` I get this
```root@lebnuc:~/obsidian-livesync# docker ps
CONTAINER ID   IMAGE                              COMMAND                  CREATED       STATUS                          PORTS      NAMES
04842a0ed291   couchdb                            "tini -- /docker-ent…"   3 days ago    Restarting (1) 54 seconds ago              obsidian-livesync
```

My `docker-compose.yml` file
```version: "2.1"
services:
  couchdb:
    image: couchdb
    container_name: obsidian-livesync
    user: 1000:1000
    environment:
      - COUCHDB_USER={my username}
      - COUCHDB_PASSWORD={my password}
    volumes:
      - ./data:/obsidian-livesync/data
      - ./local.ini:/obsidian-livesync/local.ini
    ports:
      - 5984:5984
    restart: unless-stopped
```

With the `local.ini` file being
```[couchdb]
single_node=true
max_document_size = 50000000

[chttpd]
require_valid_user = true
max_http_request_size = 4294967296
enable_cors = true

[chttpd_auth]
require_valid_user = true
authentication_redirect = /_utils/session.html

[httpd]
WWW-Authenticate = Basic realm="couchdb"
bind_address = 0.0.0.0

[cors]
origins = app://obsidian.md, capacitor://localhost, http://localhost
credentials = true
headers = accept, authorization, content-type, origin, referer
methods = GET,PUT,POST,HEAD,DELETE
max_age = 3600

**Comment by LebTeb** [comment]
Any updates?

**Comment by codedge** [comment]
After you started your CouchDB docker container you need to create a new database. To do so, open `https://my-server:5984/_utils` in a browser. This opens the admin interface for CouchDB whichs lets you do that.

The database that you created there, you need to specify later in the plugin in Obsidian.
